components/list/src/vwc-list-item.scss

```diff
@@ -114,3 +128,16 @@ $vvd-list-item-graphic-margin: --vvd-list-item-graphic-margin;
 .mdc-deprecated-list-item__meta {
 	display: flex;
 }
+
+
+:host(:not([disabled])) {
```

Comment:
this causes the icon to be white when its not active or disabled:
![image](https://user-images.githubusercontent.com/7512792/159277441-22982fd0-5a69-45a1-bd15-dc3db2e617cd.png)